ui/tests/integration/helpers/new-line-split-test.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'vault/tests/helpers';
+import { newLineSplit } from 'core/helpers/new-line-split';
+
+module('Integration | Helper | new-line-split', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it splits the string by new line characters', async function (assert) {
+    const lines = newLineSplit(['First new line.\nSecond new line.\nThird new line.']);
+    assert.deepEqual(lines, ['First new line.', 'Second new line.', 'Third new line.']);
```

Comment:
could you throw in a couple of non-new line characters and make sure they don't somehow break anything? Some examples below:
* `<br>\n`
* `/n` (reversed forward slash)
* `\n\n` two newlines next to each other. 
* Anything else you think folks might accidentally/strangely do.

Reply:
@Monkeychip Just added more tests with your suggestions! For added context, since this is base64 encoded the br and reverse /n show as strings in custom messages.